apis/java/src/test/java/io/tiledb/libvcfnative/VCFReaderTest.java

```diff
@@ -92,6 +93,122 @@ public void testReadCompletes() throws IOException {
     Assert.assertEquals(results, TOTAL_EXPECTED_RECORDS);
   }
 
+  /**
+   * Tests validity of the reader values
+   *
+   * @throws IOException
+   */
+  @Test
+  public void testReaderValues() throws IOException {
+    VCFReader reader = getVFCReader(Optional.empty(), Optional.empty());
+
+    reader.setRanges("1:12100-13360,1:13500-17350".split(","));
+
+    String[] attributes =
+        new String[] {
+          "pos_start",
+          "pos_end",
+          "sample_name",
+          "fmt_MIN_DP",
+          "fmt_SB",
+          "fmt_AD",
+          "fmt_GQ",
+          "fmt_GT",
+          "fmt_PL",
+          "fmt_DP",
+          "info_END",
+          "info_DP",
+          "info_ReadPosRankSum",
+          "info_ClippingRankSum"
+        };
+    // Set the buffers
+    for (String key : attributes) {
+      reader.setBuffer(key, ByteBuffer.allocateDirect(1024).order(ByteOrder.nativeOrder()));
+    }
+
+    reader.setBufferOffsets(
+        "sample_name", ByteBuffer.allocateDirect(1024).order(ByteOrder.nativeOrder()));
+
+    reader.setBufferOffsets(
+        "info_END", ByteBuffer.allocateDirect(1024).order(ByteOrder.nativeOrder()));
+
+    reader.setBufferValidityBitmap(
```

Comment:
Does any other field need validity buffers?

Reply:
I think that we should get advice from @aaronwolen for that, but for now we do not use it in the unit test even for the `info_END` attribute, this is a leftover from some test I've been doing during the development